torchci/rockset/commons/__sql/recent_pr_workflows_query.sql

```diff
@@ -15,32 +16,30 @@ WITH recent_shas as (
         and p.base.repo.full_name = 'pytorch/pytorch'
 )
 SELECT
+    w.id as workflow_id,
     j.id,
     j.name,
     j.conclusion,
     j.completed_at,
     j.html_url,
-    p.number AS pr_number,
-    p.head.sha as head_sha,
+    recent_shas.number AS pr_number,
+    recent_shas.sha as head_sha,
     j.torchci_classification.captures as failure_captures,
 FROM
     recent_shas
     join commons.workflow_job j ON j.head_sha = recent_shas.sha
-    left outer join commons.pull_request p on p.head.sha = j.head_sha
+    join commons.workflow_run w on w.id = j.run_id
 UNION
 SELECT
+    null as workflow_name,
```

Comment:
why?

Reply:
union requires that the number of fields be the same.  This is the part of the query that finds the workflows and the name of the workflow is stored in the name field